### ISSUE TITLE
home:ungoogled_chromium key filename breaks apt-key for ubunut / LinuxMint

### ISSUE THREAD

**Issue by DarkRF** [Eloston/ungoogled-chromium#1099]
**The issue**
The home:ungoogled_chromium install steps break the apt-key mechanism.
The issue is the colon in the name / filename. 
Issue persist on LinuxMint 20 following  the procedures on the page:
https://software.opensuse.org/download/package?package=ungoogled-chromium&project=home:ungoogled_chromium

It might prevent updates of software.
_**Error not detectable until you try to add another apt key.**_ 

**To Reproduce**
After following the steps in the install page and installed ug.-chromium to Linux Mint 20:
https://software.opensuse.org/download/package?package=ungoogled-chromium&project=home:ungoogled_chromium

a few days later I have tried to install another repo: 
https://sourceforge.net/p/ubuntuzilla/wiki/Main_Page/#installation
( failed line: `sudo apt-key adv --recv-keys --keyserver keyserver.ubuntu.com 2667CA5C` )

_result (snip):_
```
...
gpg: invalid key resource URL '/etc/apt/trusted.gpg.d/home:ungoogled_chromium.gpg'
gpg: keyblock resource '(null)': General error
gpg: key 02456C79B2FD48BF: 1 signature not checked due to a missing key
...
```

All the signatures after were failing.

I have removed the file with the semicolon.
I went back to the ungoogled-chromium install page and simply used the filename without the ":" colon character. This workaround got the whole thing working and after an apt-update I have recognized that there are new updates for ungoogled chromium as well.
`curl -fsSL https://download.opensuse.org/repositories/home:ungoogled_chromium/Ubuntu_Focal/Release.key | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/ungoogled_chromium.gpg > /dev/null`

**Expected behavior**
I expect that your install procedures do not break my system.
Yes, it looks like a gpg but mainly due to the filenames.  
It would be still good if you could fix the install procedure and use filenames without special characters like semicolons in it. 
At least for the time being while you convince the Ubuntu, apt-key and/or gpg devs to fix the issue.

**Environment (please complete the following information):**
 - OS/Platform and version:  
Linux Mint 20 Ulyana

 - ungoogled-chromium version:
N/A - not version specific, procedure specific related to install to Ubuntu from repo.

**Additional context**
Not sure if it blocks updates but I have recognized a new version right after applying the workaround.
(Version 83.0.4103.116 (Developer Build) built on Ubuntu focal/20.04, running on Linuxmint ulyana/20 (64-bit))
83.0.4103.116.focal1 to 83.0.4103.116.focal2
Maybe just a coincidence.

_Additional notes:_
Please, list the codes not only the names of the Ubunut versions on the install page. It is quite an extra step to figure out without the numbers which Ubuntu version is which and not everyone knows it by heart. As an extra, you could add in the parentheses Linux Mint or other major distros as well.
[ example: "concupiscent coati (69.04)(LTS)" instead just writing "concupiscent" ]
Other hand: Keep up the good work!

**Comment by Fallast** [comment]
i think my problem is related to this

**Comment by andreasmjg** [comment]
Hi - thank you, this helped me as well. I had the same problem on Ubuntu Bionic (18.04 LTS) following the instructions on the OpenSuse production side for Ubuntu here: https://software.opensuse.org/download/package?package=ungoogled-chromium&project=home:ungoogled_chromium

This solved it: 

1) Remove the key with the : in the filename:

`sudo rm /etc/apt/trusted.gpg.d/home\:ungoogled_chromium.gpg`

2) Re-download the key from OpenSuse but with the different name when placed on the system: 

`curl -fsSL https://download.opensuse.org/repositories/home:ungoogled_chromium/Ubuntu_Bionic/Release.key | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/ungoogled_chromium.gpg > /dev/null`

**Comment by Eloston** [comment]
Hmm, it seems like OBS continues to have problematic instructions.

@braewoods Have you run into this problem?

**Comment by braewoods** [comment]
@Eloston No but I think the basic problems boil down to inexperienced users. We're probably going to have to create our own because people keep asking about or reporting issues we already know about.

**Comment by jstkdng** [comment]
Wouldnt it be better to download the binaries from the OBS and upload them to launchpad maybe?
That is what I do with my repo.

**Comment by braewoods** [comment]
You can't do that. Launchpad only takes source packages and I abandoned it because of its own problems that OBS is actually better at.

**Comment by jstkdng** [comment]
Hmm, well, OBS is really good at building packages, but it is really bad at managing repositories.
I could donate some space on my vps if you need it @braewoods .

**Comment by Eloston** [comment]
OBS seems to be fine at hosting repositories, but it has a bad UX. Writing our own instructions would solve these problems. If we wanted to get fancy with the instructions, we could add GitHub Pages to the debian repo.

**Comment by DarkRF** [comment]
Possible solutions:

- Report a bug or fix apt-key for long term.
- Report a bug to fix the automation of your site.
- Write own install instructions.
  - Plus: Write a wrapper script to handle the management of the repos / install sources.

The infrastructure if you switch might have other issues. This situation causing pesky hidden problems for end users and can negativity impact project reputation. Please, try to pursue an immediate fix and trigger long term solutions with the upstream tools.

Apologies for being pushy and might be arrogant but I have been in similar cases and several times the outcome was not that good.

I also understand that hosting solutions might be considered but please try to discuss it separately and refer only milestones/links here. It was hard to follow and got confused reading through.

**Comment by nicolaasjan** [comment]
It seems OBS is working on it.
https://github.com/openSUSE/software-o-o/issues/842

**Comment by Eloston** [comment]
Gonna wait for upstream to fix before closing.

**Comment by braewoods** [comment]
I'll investigate it once the new builds are finished.
